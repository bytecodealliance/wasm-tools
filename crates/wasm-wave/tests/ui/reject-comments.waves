basic-record(// { required: 1 });
basic-record({ required: 1 } // );
basic-record(// {
  required: 1,
});
basic-record(
  {
    required: 1,
//}
);
basic-record({
  // required: 1,
  optional: none,
});