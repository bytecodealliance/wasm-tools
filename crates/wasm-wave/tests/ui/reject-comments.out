unexpected end of input at 32..32
unexpected end of input at 33..33
unexpected token: Colon at 28..29
unexpected token: ParenClose at 39..40
error converting Wasm value: missing field "required" at 13..52
